gradle.properties

```diff
@@ -1,8 +1,6 @@
 version=1.0.10
 org.gradle.parallel=true
 spinnakerGradleVersion=8.10.0
-pf4jVersion=3.2.0
-korkVersion=7.59.0
+korkVersion=7.142.0
 orcaVersion=8.10.0
```

Comment:
Sorry for just realizing this now...but orca has also moved to [io.spinnaker.orca](https://repo.maven.apache.org/maven2/io/spinnaker/orca/) and since it brings in kork, it seems like this code here would be better off not specifying the version of kork...

but now that we're down this road, I say let's keep going and specify a newer version of orca...and I guess since this is the master branch of the plugin, it makes sense to use a version of orca from its master branch as well...The latest orca version from master is [8.24.0](https://github.com/spinnaker/orca/tree/v8.24.0), but unfortunately it doesn't look like the artifacts were properly released.  Like, there's no 8.24.0 directory [here](https://repo.maven.apache.org/maven2/io/spinnaker/orca/orca-bom/), and [this job](https://github.com/spinnaker/orca/actions/runs/2138915288) failed....so, lemme try tagging the head of orca master with 8.25.0 and see what happens there...

Reply:
version 8.25.0 of orca was released successfully.

Reply:
Thank so much!, didn’t realize that was the reason i was still getting dependencies from com.netflix.kork